### ISSUE TITLE
Is this code incomplete? 

### ISSUE THREAD

**Issue by Hui-zju** [oacore/prompt_citation_classification#1]
Where is the file "cli.py"?

**Comment by suchetha88** [comment]
Yes, cli.py was missing. Thanks for pointing it out. This is now added. Btw, script for pet is adapted from https://github.com/TevenLeScao/pet/tree/master
